### ISSUE TITLE
firehol.org/news/ says iprange 1.0.5 but there is only 1.0.4

### ISSUE THREAD

**Issue by SanskritFritz** [firehol/iprange#20]
My question is, was it a typo or there is a release missing?

**Comment by ktsaou** [comment]
hm... probably a typo.
@philwhineray ?

**Comment by philwhineray** [comment]
Yeah, sorry, not sure what I did there. The master branch looks like it was
updated to 1.0.5, in my local repo at least, but there is no tag.

There isn't any code changes subsequent to 1.0.4 so you definitely aren't
missing anything.

I'll try to work out what I was playing at and will finish it if necessary,
or more likely revert the news item.

Off the top of my head it was probably a change to get releases to appear
as a generic firehol user but I abandoned it when I discovered there is
nothing updated in git.
